client/my-sites/email/titan-set-up-thank-you/index.tsx

```diff
@@ -13,20 +14,23 @@ import {
 import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
 import getCurrentRoute from 'calypso/state/selectors/get-current-route';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type { Argument } from 'classnames';
```

Comment:
It's first time I see this type!

For reference, this type is a compound type described [here](https://github.com/JedWatson/classnames/blob/94acb8701bd00c4cc4f7c30a4acea8408b0dcd30/index.d.ts)

Reply:
My question is, is it really worthy? One of the types that include Argument is `string`. So I wonder, for readability purposes, is it better to use `Argument` or `string`?

In my case, I had to check what the `Argument` is, which added a bit of complexity to the process of understanding the code.

I'm leaving this on your plate, as the type is correct, but we try not to add unnecessary code, and seems to me that string type is enough, and even future prof.

Reply:
I may have been overly diligent here. It doesn't add any weight to the built JS bundle since it's only a type annotation, but sure, we can keep it simple! It's dubious that we will need to specify anything other than a string for this prop in the future